domains/auth/Auth.API/API/Services/SignaturGruppen/ISignaturGruppen.cs

```diff
@@ -0,0 +1,7 @@
+using API.Models;
+
+namespace API.Services;
+public interface ISignaturGruppen
+{
+    LoginResponse CreateRedirecthUrl(string feUrl, string returnUrl);
```

Comment:
I would change this to GetNext(AuthState state)

and then create a new empty state based on these parameters in the login controller to start the flow, then we would already have the basis for the orchestrator done.